switchlink/testing.cmake

```diff
@@ -117,6 +117,9 @@ set_target_properties(krnlmon-test PROPERTIES EXCLUDE_FROM_ALL TRUE)
 add_custom_target(krnlmon-coverage
     lcov --capture --directory ${CMAKE_BINARY_DIR}
     --output-file krnlmon.info
+    --exclude '/opt/deps/*'
+    --exclude '/usr/include/*'
+    --exclude '9/**'
```

Comment:
What is the `9/**` directory? That seems odd

Reply:
It's part of the C++ library infrastructure.

We does not like it, my precious, because we only uses part of it, and that drags our code coverage downward:

![krnlmon-coverage-old](https://github.com/ipdk-io/krnlmon/assets/12834857/c2325b95-20d9-4b64-8871-6f0b9ae30c7b)

The picture becomes rosier (and more accurate) when we tell `lcov` to ignore these header files:

![krnlmon-coverage-new](https://github.com/ipdk-io/krnlmon/assets/12834857/64b0cd53-d3dc-4892-ac3f-f257c637dd74)

Reply:
To see this for yourself:

```bash
cmake -B build -DTDI_TARGET=es2k -DWITH_OVSP4RT=off -DTEST_COVERAGE=on
cmake --build build -j4 --target krnlmon-test krnlmon-coverage
```

Then use a browser to view `build/Testing/coverage/index.html`.